src/App.js

```diff
@@ -1,25 +1,120 @@
-import logo from './logo.svg';
-import './App.css';
+import logo from "./logo.svg";
+import "./App.css";
+import {useEffect, useState} from "react";
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {faTrashCan} from "@fortawesome/free-regular-svg-icons";
 
-function App() {
+function NavBar() {
+  return (
+    <nav
+      className="flex flex-wrap justify-center items-center gap-4 border-b-2 border-b-gray-200 px-7 py-3 font-display">
+      <a href="/">
+        <img src={logo} width="40" alt="logo"/>
+      </a>
+      <a className="text-4xl font-extrabold text-primary" href="/">tomo</a>
+      <span className="sm:flex-grow"></span>
+      <div className="block md:inline">
+        <a className="text-2xl text-neutral-gray font-bold hover:text-primary" href="/classes">Classes</a>
```

Comment:
optional, but instead of passing in hardcoded strings like "Classes", you can pass in props or use separate config files to make it easier to make changes to these values. this would update them everywhere they are used.